### PR TITLE
Fix error messages in bleio

### PR DESCRIPTION
**Pull request by deshipu**

### CODE REVIEW

**Comment by dhalbert** [comment]
Travis failed due to translations in the first subjob.

**Comment by deshipu** [comment]
Sorry, forgot to run make translate again.

File: ports/nrf/common-hal/bleio/Characteristic.c [hunk 1]
```diff
@@ -129,7 +129,7 @@ STATIC void gatts_notify_indicate(bleio_characteristic_obj_t *characteristic, mp
     const uint32_t err_code = sd_ble_gatts_hvx(conn_handle, &hvx_params);
     if (err_code != NRF_SUCCESS) {
         m_tx_in_progress--;
-        mp_raise_OSError_msg_varg(translate("Failed to notify or indicate attribute value, err %0x04x"), err_code);
+        mp_raise_OSError_msg_varg(translate("Failed to notify or indicate attribute value, err x0%04x"), err_code);
     }
```
Comment (jerryneedell) [inline ports/nrf/common-hal/bleio/Characteristic.c]: shouldn't this be 0x%04x ?

File: ports/nrf/common-hal/bleio/Characteristic.c [hunk 2]
```diff
@@ -141,7 +141,7 @@ STATIC void gattc_read(bleio_characteristic_obj_t *characteristic) {
 
     const uint32_t err_code = sd_ble_gattc_read(conn_handle, characteristic->handle, 0);
     if (err_code != NRF_SUCCESS) {
-        mp_raise_OSError_msg_varg(translate("Failed to read attribute value, err %0x04x"), err_code);
+        mp_raise_OSError_msg_varg(translate("Failed to read attribute value, err x0%04x"), err_code);
     }
```
Comment (jerryneedell) [inline ports/nrf/common-hal/bleio/Characteristic.c]: same as above 0x%04x

**Comment by dhalbert** [comment]
Good point, @jerryneedell. I should have looked more carefully. @deshipu could you submit another PR?

**Comment by deshipu** [comment]
Sure, sorry about that. Looks like those messages are cursed...

**Comment by deshipu** [comment]
Done in #1699

**Pull request merged**